docs/release-notes.md

```diff
@@ -12,6 +12,9 @@ nav_order: 9
 
 ### Features
 
+- Stabilize OpenShift spec 4.13.0, targeting Ignition spec 3.2.0
+- Add OpenShift spec 4.14.0-experimental, targeting Ignition spec
+  3.4.0
 - Allow enabling discard passthrough on LUKS devices _(fcos 1.5.0-exp,
```

Comment:
The `4.13.0-exp` release notes all need update to `4.13.0` or `4.14.0-exp`, depending on the item.

Reply:
Oh, ok

Reply:
So should I update the 3.4.0-exp notes to 3.4.0 as well while I am here? 
noticed they were still calling it exp